### ISSUE TITLE
[BUG] Plugin breaking after updating to 3.0.0

### ISSUE THREAD

**Issue by mjknght** [franciskafieh/obsidian-list-modified#111]
Hi! Forgive me if I am missing something, but I can't find any instructions regarding where the output format templates should be placed in the new version of the plugin. I have been attempting to place them like so, but this does not modify the output:

```
%% LIST CREATED %%
- [[name]], [[ctime]]
%% END %%
```

Thanks for this plugin, and in advance for your guidance on the new output format templating!

**Comment by franciskafieh** [comment]
Hi there. In the output format settings in the List Modified tab, set your
output format to [[name]], [[ctime]]. you should never have to put stuff in
between the dividers yourself, that’s what the plugin does. I should
probably add this to the wiki to make it clearer. I will do it when I get
on my laptop

**Comment by mjknght** [comment]
Hi, thanks for a quick response! What you have suggested was my first thought to, but with plugin version 3.1.3 I don’t seem to see the output format setting? These are the only settings I have:

<img width="885" alt="Screenshot 2024-10-16 at 11 21 37 AM" src="https://github.com/user-attachments/assets/9f31d981-1210-4f46-876d-2c0118510cb7">

**Comment by franciskafieh** [comment]
Would you be able to send me your `data.json` for the plugin? This may be a migration issue when moving to 3.1.3

To access the data.json, navigate to your vault folder, go into the `.obsidian` hidden folder > `plugins` > `obsidian-list-modified` and copy and paste the content of `data.json`.

It should look something like this:
![image](https://github.com/user-attachments/assets/ed4173ec-29fa-445c-b94f-12b26424051a)

feel free to delete the trackedFiles section to hide personal info.

**Comment by franciskafieh** [comment]
Also, do you recall seeing a popup saying something like "migrating to 3.x.x" when you updated the plugin? This stuff should have been done automatically, and it did work when I tested it awhile ago.

**Comment by mjknght** [comment]
Afraid I don't recall seeing the migration popup! But here's the JSON

```
{
  "excludedTags": [],
  "excludedFolders": "Templates, BearImages",
  "excludedNameContains": [],
  "outputFormat": "- [[path]]",
  "separateOutputFormats": false,
  "createdFormat": "- [[link]]",
  "modifiedFormat": "- [[link]]",
  "deletedFormat": "- [[path]]",
  "timeFormat": "YYYY-MM-DD",
  "autoCreateLogNote": false,
  "logNoteType": "daily",
  "writeInterval": 0,
  "combineCreatedAndModified": false,
  "autoCreateCreatedDivider": false,
  "autoCreateModifiedDivider": false,
  "autoCreateDeletedDivider": false,
  "verboseModeEnabled": false,
  "lastTrackedDate": "2024-10-16",
  "trackedFiles": [
    {
      "path": "9 October 2024.md",
      "matchesCriteria": true,
      "supposedList": "modified"
    },
    {
      "path": "Friedrich Nietzsche’s discussion of ignorance.md",
      "matchesCriteria": true,
      "supposedList": "modified"
    },
    {
      "path": "Christopher Kaiser.md",
      "matchesCriteria": true,
      "supposedList": "modified"
    },
    {
      "path": "<9.md",
      "supposedList": "created",
      "matchesCriteria": true
    },
    {
      "path": "\"9.md",
      "supposedList": "created",
      "matchesCriteria": true
    },
    {
      "path": "9.md",
      "supposedList": "created",
      "matchesCriteria": true
    },
    {
      "path": "1 Timothy 3.md",
      "matchesCriteria": true,
      "supposedList": "modified"
    }
  ],
  "tags": "",
  "ignoredNameContains": "",
  "appendSpaceAfterHeadings": false,
  "autoCreatePrimaryHeading": false,
  "primaryHeading": "Notes",
  "modifiedHeading": "Updated",
  "separateCreated": true,
  "createdHeading": "New",
  "separateDeleted": false,
  "deletedHeading": "Deleted"
}
```

**Comment by franciskafieh** [comment]
That’s odd. As a temporary solution for now, feel free to close Obsidian,
then change the data.json outputFormat manually to your desired format.
Once you reopen Obsidian, the plugin should respect your settings. I will
have to look into this more tomorrow. Thanks for the report.

**Comment by mjknght** [comment]
No worries! I had not used the plugin to any great extent, so I uninstalled and reinstalled just now, and can see all the settings. This is a great plugin, thanks so much for sharing it.

**Comment by franciskafieh** [comment]
I’m glad you like it!

Did you have the plugin installed previously though? Or did you just
download it after 3.1.3? From your data.json, the old settings seem to be
present still, so I would assume an upgrade? Just wondering because it
would make fixing this a lot easier.

**Comment by mjknght** [comment]
Yes, I had it installed, so it was indeed an upgrade. Hope that’s useful information! I just had only tinkered with the plugin before – wasn’t in regular use, so didn’t need to think twice about deleting and reinstalling.

**Comment by franciskafieh** [comment]
Migration works fine for me from 2.x.x to 3.x.x. UI elements render normally. Maybe something bugged out for you. This is likely since nobody else has reported having this issue. If anyone else does experience this issue, though, please re-open

**Comment by franciskafieh** [comment]
Re-opening since other users have reported the same issue...

**Comment by franciskafieh** [comment]
fixed on my end, but please re-open if this is not fixed for anyone.

**Comment by joh6nn** [comment]
Hey boss, it's me again 😅

I hate to be the bearer of bad news, but this is still happening for me on v3.4.0, on both Android and Desktop.

Reinstalling temporarily fixes it and then it breaks a little while later. I haven't nailed down what causes the breakage yet. Will collect as much info as I can and report back. If there's anything in particular you want me to look for, just let me know.

Belated happy holidays and hope you're doing well!

**Comment by franciskafieh** [comment]
> this is still happening for me

as in, some settings disappearing?

**Comment by joh6nn** [comment]
yup. And the issue from #115 as well, though I suspect that they're just different symptoms of the same issue.

Desktop: Obsidian 1.7.7, installer 1.6.3
Mobile: Obsidian 1.7.7

**Comment by franciskafieh** [comment]
Have you tried deleting the entire .obsidian/plugins/obsidian-list-modified folder on all devices, then re-installing the plugin (keep in mind - this will reset your note lists for the current period)?

**Comment by franciskafieh** [comment]
Going to close again for now. Feel free to re-open if any new issues/insights.
